js/phemex.js

```diff
@@ -55,13 +63,17 @@ module.exports = class phemex extends Exchange {
                 'fetchOrderBook': true,
                 'fetchOrders': true,
                 'fetchPositions': true,
+                'fetchPositionsRisk': false,
                 'fetchPremiumIndexOHLCV': false,
                 'fetchTicker': true,
                 'fetchTrades': true,
                 'fetchTradingFee': false,
                 'fetchTradingFees': false,
                 'fetchWithdrawals': true,
+                'reduceMargin': undefined,
                 'setLeverage': true,
+                'setMarginMode': false,
```

Comment:
do you know how to place a cross/isolated order on phemex? I couldn't find anything about it in their api docs

Reply:
The docs don't have many details but using the exchange I don't think you're allowed to set the leverage for cross on Phemex it's just a predetermined amount like for btc its 0.83x and if you set the leverage to anything else it seems to make it an isolated position

Reply:
Yeah that's how it works https://phemex.com/user-guides/how-do-i-adjust-the-margin-on-my-position